test_util/test_runner.py

```diff
@@ -19,7 +19,7 @@ def integration_test(
         dcos_dns, master_list, agent_list, public_agent_list,
         test_dns_search, provider,
         aws_access_key_id='', aws_secret_access_key='', region='', add_env=None,
-        pytest_cmd='py.test -vv',
+        pytest_cmd='py.test -rs -vv',
```

Comment:
Would prefer to keep it as part of this PR, since it is in a separate commit and part of this change is to intentionally skip certain tests if the cluster was configured with telemetry disabled (new behavior). So increasing the visibility of skip test details in our logs seems valuable. However I could move it to a separate PR if you feel strongly about it.